Games/core/Meeting.js

```diff
@@ -638,7 +638,8 @@ module.exports = class Meeting {
                     (!this.multi && this.votes[member.id] == null) ||
                     (this.multi && selections.length < this.multiMin && selections.indexOf("*") == -1)
                 ) {
-                    this.game.vegPlayer(member.player);
+                    const playerIsKicklynched = this.actionName === "Village Vote" && this.finalTarget === member.id;

```

Comment:
This would solve the exe issue

I'm not sure if it should record abandons if it was doing to by voted off...
```
                    const isKickEliminated = this.actionName === "Village Vote" && this.finalTarget === member.id;
                    if (isKickEliminated) {
                        this.game.vegPlayer(member.player);
                    } else {
                        member.player.recordStat("abandons", true);
                    }
```